simplecpp.cpp

```diff
@@ -888,7 +886,7 @@ void simplecpp::TokenList::readfile(Stream &stream, const std::string &filename,
             }
 
             if (prefix.empty())
-                push_back(new Token(s, location)); // push string without newlines
+                push_back(new Token(s, location, isspace(stream.peekChar()))); // push string without newlines
```

Comment:
I believe this is not guaranteed to be portable. we include the `<cctype>` so we should call `std::isspace`.